docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1700 @@
+{
```

Comment:
<p>Since STUMPY now supports p-norm (in addition to non-normalized Euclidean distance - i.e., <code>p=2</code> ), what would be the maximum <code>min_dist</code>  when, say, <code>p=1</code> or <code>p=3</code>?   For p-norm, is the maximum <code>min_dist</code> :</p><p> </p><pre class="ql-syntax" spellcheck="false">np.abs(np.nanmax(T) - np.nanmin(T)) * (m ** (1.0 / p)) 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I think that is correct. According to the definition provided in [WikiPedia](https://en.wikipedia.org/wiki/Norm_(mathematics)) and using similar approach provided in the notebook, I got the same value.

Reply:
<p>Thank you!</p>